content/locales/en.ts

```diff
@@ -172,5 +172,10 @@ export const en = {
       "{characterCount, plural, one {1 character} other {{characterCount} characters}}",
     somethingWrong: "Something went wrong. Please try again later.",
     invalidValidationCode: "Invalid validation code.",
+    identityInputCantBeBlank: "Please fill in the following field.",
```

Comment:
```suggestion
    identityInputCantBeBlank: "Please fill in the following field: ",
```

It's just a suggestion, I'm not sure what's the best between the two (same goes for the corresponding key in fr translations)

Reply:
I made the changes :).